aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -172,6 +294,30 @@ module aptos_framework::staking_config {
         staking_config.rewards_rate_denominator = new_rewards_rate_denominator;
     }
 
+    public fun update_rewards_config(
+        aptos_framework: &signer,
+        rewards_rate: FixedPoint64,
+        min_rewards_rate: FixedPoint64,
+        rewards_rate_period_in_secs: u64,
+        rewards_rate_decrease_rate: FixedPoint64,
+    ) acquires StakingRewardsConfig {
+        assert!(features::reward_rate_decrease_enabled(), error::invalid_state(EDISABLED_FUNCTION));
```

Comment:
I think we should remove this in case we mis-initialize and need to call this to update before the feature is enabled